pages/linux/minicom.md

```diff
@@ -0,0 +1,16 @@
+# minicom
+
+> A program to do a serial communication with a device.
+> More information: <https://manned.org/minicom>.
+
+- Open a given serial port:
+
+`sudo minicom --device {{/dev/ttyUSB0}}`
+
+- Open a given serial port with a given baud rate:
+
+`sudo minicom --device {{/dev/ttyUSB0}} --baudrate {{115200}}`
+
+- Enter setup page before doing communication with a given serial port:
```

Comment:
Perhaps this will read better? The language `configuration menu` is used in the official man page.
```suggestion
- Enter the configuration menu before communicating with a given serial port:
```

Reply:
Yup, this looks better.